plugin/pkg/global/extensionlabels/admission.go

```diff
@@ -242,6 +242,8 @@ func addMetaDataLabelsSeed(seed *core.Seed) {
 	if seed.Spec.DNS.Provider != nil {
 		metav1.SetMetaDataLabel(&seed.ObjectMeta, v1beta1constants.LabelExtensionDNSRecordTypePrefix+seed.Spec.DNS.Provider.Type, "true")
 	}
+	metav1.SetMetaDataLabel(&seed.ObjectMeta, v1beta1constants.LabelSeedProvider, seed.Spec.Provider.Type)
+	metav1.SetMetaDataLabel(&seed.ObjectMeta, v1beta1constants.LabelSeedRegion, seed.Spec.Provider.Region)
```

Comment:
Is this the right place to add these labels? If you check https://github.com/gardener/gardener/blob/v1.124.0/docs/concepts/apiserver-admission-plugins.md#extensionlabels, this admission plugin maintains only extension specific labels. These labels here are not extension specific.